pymongo/asynchronous/mongo_client.py

```diff
@@ -861,6 +861,10 @@ def __init__(
             # This will be used later if we fork.
             AsyncMongoClient._clients[self._topology._topology_id] = self
 
+    async def connect(self) -> None:
```

Comment:
"Connect" might not be the best name here since we're not actually connecting. I know we have the "connect=True" argument but I can't help but feel like "open()" would be more appropriate. If a user actually wanted to connect I would expect them to run something like "db.command('ping')". What do you think?

Reply:
It feels a bit odd to me for the synchronous API to have a `connect=True` option but the equivalent async option is called `open` instead even though they do the same thing. Consistency of naming makes more sense to me here.

Reply:
In that case I think the docstring just needs to explain the concept more clearly.